common/scala/src/main/scala/org/apache/openwhisk/core/mesos/MesosContainerFactory.scala

```diff
@@ -126,7 +126,7 @@ class MesosContainerFactory(config: WhiskConfig,
                                memory: ByteSize,
                                cpuShares: Int)(implicit config: WhiskConfig, logging: Logging): Future[Container] = {
     implicit val transid = tid
-    val image = if (userProvidedImage) {
+    val image = if (userProvidedImage && !runtimesRegistryConfig.includeUserImages.getOrElse(false)) {
```

Comment:
This pattern is repeating itself in 4 impls so far. May be we should move this logic to `ContainerFactory` object. Or introduce a new `RuntimesRegistry` object and use that 

```scala
object RuntimesRegistry {
  def resolveImage(actionImage: ExecManifest.ImageName,
                   userProvidedImage: Boolean,
                   config: RuntimesRegistryConfig): Either[ExecManifest.ImageName, String] = {
    if (userProvidedImage && !config.includeUserImages.getOrElse(false)) {
      Left(actionImage)
    } else {
      Right(actionImage.localImageName(config.url))
    }
  }
}
```